eve/main.yml

```diff
@@ -37,4 +37,4 @@ stages:
           workdir: '%(prop:builddir)s/build'
       - ShellCommand:
           name: run feature tests
-          command: npm run ft_test
+          command: TEST_SWITCH=1 npm run ft_test
```

Comment:
You can also add `TEST_SWITCH` to the environment.
Something like
```
       - ShellCommand:
            name: run feature tests
            command: npm run ft_test
            env:
              TEST_SWITCH: "1"
```

Reply:
Actually the way I did it matches what has been introduced in development/8.0 branch. I can nevertheless do this change also on the integration branch.